moduleroot/.github/workflows/test.yml.erb

```diff
@@ -8,6 +8,17 @@ env:
   BUNDLE_WITHOUT: release
 
 jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.0'
+          bundler-cache: true
+      - name: Run rake rubocop
+        run: bundle exec rake rubocop
   test:
```

Comment:
Could you make this depend on rubocop? That may save some CI in case a style is wrong.